lib/trento_web/openapi/v1/schema/user.ex

```diff
@@ -6,6 +6,71 @@ defmodule TrentoWeb.OpenApi.V1.Schema.User do
 
   alias TrentoWeb.OpenApi.V1.Schema.Ability.AbilityCollection
 
+  defmodule UserTOTPEnrollmentPayload do
+    @moduledoc false
+
+    @schema %Schema{
+      title: "UserTOTPEnrollmentPayload",
+      description: "Trento User totp enrollment payload",
```

Comment:
Let's put `topt `as upper case `TOTP` as in other texts

Reply:
done!